### PR TITLE
Style Guide: Added rule for do-while loops

### PR DESCRIPTION
**Pull request by Stevoisiak**

### CODE REVIEW

**Comment by phire** [comment]
lgtm

**Comment by phire** [comment]
Er, actually that doesn't render in markdown correctly.

It puts it all on the same line (```do { } while (false);```) which is not something we want to recommend.

File: Contributing.md [hunk 1]
```diff
@@ -159,6 +159,12 @@ private:
 - Empty-bodied loops should use braces after their header, not a semicolon.
   - Yes: `while (condition) {}`
   - No: `while (condition);`
+- For do-while loops, place 'while' on the same line as the closing brackets
+  ```c++
```
Comment (lioncash) [inline Contributing.md]: Put a new line in between the sentence and the code block, otherwise it'll be formatted wrong.

**Comment by Stevoisiak** [comment]
Should be good now. Seems like there was an issue with my markup viewer before.

**Pull request merged**